src/main/java/org/opentripplanner/gtfs/GtfsLibrary.java

```diff
@@ -94,24 +96,25 @@ public static TraverseMode getTraverseMode(Route route) {
         }
         /* Original GTFS route types. Should these be checked before TPEG types? */
         switch (routeType) {
-        case 0:
-            return TraverseMode.TRAM;
-        case 1:
-            return TraverseMode.SUBWAY;
-        case 2:
-            return TraverseMode.RAIL;
-        case 3:
-            return TraverseMode.BUS;
-        case 4:
-            return TraverseMode.FERRY;
-        case 5:
-            return TraverseMode.CABLE_CAR;
-        case 6:
-            return TraverseMode.GONDOLA;
-        case 7:
-            return TraverseMode.FUNICULAR;
-        default:
-            throw new IllegalArgumentException("unknown gtfs route type " + routeType);
+            case 0:
+                return TraverseMode.TRAM;
+            case 1:
+                return TraverseMode.SUBWAY;
+            case 2:
+                return TraverseMode.RAIL;
+            case 3:
+            case 11:
```

Comment:
please at least give a comment, that it is a `TROLEJBUS` but we will treat it as a `BUS`